Tests.Unit.Infrastructure/packages.config

```diff
@@ -0,0 +1,10 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="xunit" version="2.4.1" targetFramework="net461" />
```

Comment:
@strongmindsmhs  Det er en anden version end resten af projekterne.. jeg er usikker på om det vil virke på byggeren, men lad os helst køre på samme version

Reply:
![image](https://user-images.githubusercontent.com/12400733/72833857-f791cb00-3c87-11ea-8aaf-400cfe1922ea.png)

Reply:
@mrjsawdk Har nu rullet tilbage til 2.1.0 for alle projekter (mener ikke jeg har opdateret for Integration tests)

Reply:
@mrjsawdk Nah, det går ikke .. 2.4.1 har været i brug længe på integrationtests (web) .. måske et krav fra protractor? Jeg ruller tilbage på 2.4.1 på dette projekt (og måske bør vi opgradere de andre projekter tilsvarende?)

Reply:
@mrjsawdk Det bøvler en del .. kan se 2.4.1 er indført 01-08-2019 på Presentation.Web test projektet. Min R# vælger hele tiden den forkerte udgave og opdaterer nuget fil, men ikke csproj

Reply:
@mrjsawdk Nu bygger og kører tests på buildserver (såvidt jeg kan se) .. men vi skal nok lige kigge på det imorgen

Reply:
@strongmindsmhs FYI: Protractor har ikke noget med C# at gøre ;-).. men ja lad os løse det sammen